File: src/main/java/com/gangofconnectfour/powerfourservice/repository/ScoreLoggerRepository.java
package com.gangofconnectfour.powerfourservice.repository;

import com.gangofconnectfour.powerfourservice.model.ScoreLogger;
import org.springframework.data.jpa.repository.JpaRepository;
import org.springframework.stereotype.Repository;

@Repository
public interface ScoreLoggerRepository extends JpaRepository<ScoreLogger, Long> {
}

File: src/main/kotlin/com/gangofconnectfour/powerfourservice/repository/ScoreLoggerRepository.kt
package com.gangofconnectfour.powerfourservice.repository

import com.gangofconnectfour.powerfourservice.model.ScoreLogger
import org.springframework.data.jpa.repository.JpaRepository
import org.springframework.stereotype.Repository

@Repository
interface ScoreLoggerRepository : JpaRepository<ScoreLogger, String> {
}